src/analysis/exploration/printRxnFormula.m

```diff
@@ -158,105 +153,69 @@
         formulaStr = '';
         for j = 1:length(reactMets)
             if (j > 1)
-                if (printFlag)
-                    fprintf(fid, '+ ');
-                end
-                formulaStr = [formulaStr '+ '];
+                formulaStr = sprintf('%s+ ', formulaStr);
```

Comment:
`formulaStr` should only be built if `printFlag` is set.

Reply:
This comment doesn't seem to make a lot of sense. In the original, fomulaStr was built always, since it is one of the return values.
In my program, it is built, and used for both printing and return, to reduce code duplication.

Reply:
Sorry, I didn't see that `formulaStr` is actually returned in `formulas`. 😄